lib/poxa/web_hook/event_table.ex

```diff
@@ -0,0 +1,31 @@
+defmodule Poxa.WebHook.EventTable do
+  @table_name :web_hook_events
+
+  def new do
+    :ets.new(@table_name, [:bag, :public, :named_table])
+  end
+
+  def init do
+    case :ets.info(@table_name) do
```

Comment:
Why do you need this? This is the only place where we create the ets table. It crash if the table exists already